prometheus_hardware_exporter/collector.py

```diff
@@ -733,3 +734,113 @@ def fetch(self) -> List[Payload]:
     def process(self, payloads: List[Payload], datastore: Dict[str, Payload]) -> List[Payload]:
         """Process the payload if needed."""
         return payloads
+
+
+class SsaCLICollector(BlockingCollector):
+    """Collector for storage arrays that support ssacli."""
+
+    ssacli = SsaCLI()
+
+    @property
+    def specifications(self) -> List[Specification]:
+        return [
+            Specification(
+                name="ssacli_command_success",
+                documentation="Indicates if the command is successful or not",
+                metric_class=GaugeMetricFamily,
+            ),
+            Specification(
+                name="ssacli_controllers",
+                documentation="Total number of controllers",
+                metric_class=GaugeMetricFamily,
+            ),
+            Specification(
+                name="ssacli_controller_status",
+                documentation="Indicates the status of controller part",
+                labels=["slot", "part"],
+                metric_class=GaugeMetricFamily,
```

Comment:
Please use InfoMetricFamily and the naming could be ssacli_controller_info
Sample code: https://github.com/canonical/prometheus-hardware-exporter/blob/main/prometheus_hardware_exporter/collector.py#LL52C30-L52C46